pkgs/applications/misc/hollywood/default.nix

```diff
@@ -0,0 +1,56 @@
+{ stdenv, fetchFromGitHub, makeWrapper, lib, coreutils, apg, atop, bmon, cmatrix
+, pygments, moreutils, util-linux, jp2a, man, mplayer, openssh, tree, mlocate
+, findutils, ccze, ncurses, python3, wget, libcaca, newsboat, rsstail, w3m
+, ticker, tmux, byobu }:
+let
+  pathDeps = [
+    tmux
+    coreutils
+    ncurses
+    jp2a
+    mlocate
+    apg
+    atop
+    bmon
+    cmatrix
+    pygments
+    moreutils
+    util-linux
+    jp2a
+    man
+    mplayer
+    openssh
+    tree
+    findutils
+    ccze
+  ];
+in stdenv.mkDerivation {
+  pname = "hollywood";
+  version = "1.22";
+  src = fetchFromGitHub {
+    owner = "dustinkirkland";
+    repo = "hollywood";
+    rev = "35275a68c37bbc39d8b2b0e4664a0c2f5451e5f6";
```

Comment:
Why are you using raw `rev` hashes instead of a human-readable tag like `rev = v${version}`?

Reply:
The GitHub mirror doesn’t have tags.